tests/test_views.py

```diff
@@ -74,7 +74,7 @@ def test_view_dataset_with_recommendations_dedupe(self, datasets, reuses):
 
         content = render_hook(dataset=dataset)
 
-        assert content.count(f'href="{ds1.external_url}"') == 1
+        assert content.count(f'href="{ds1.display_url}"') == 1
```

Comment:
I did not see where this modification comes from

Reply:
This is coming from [udata-front card-sm](https://github.com/etalab/udata-front/pull/181/files#diff-509bf4b5ff62f9953a8998252980fd626b73042c4dcdb984bf1c014607739695R52).

Previously, we had a card template with an external URL and a search-result template without.

Now, both are using the same link (`url_for('datasets.show', dataset=dataset)`).